public/docs/ts/latest/guide/index.jade

```diff
@@ -120,10 +113,10 @@ block example-links
 
   # Feedback
 
-  We welcome feedback! Leave a comment by clicking the icon in upper right corner of the banner.
+  We welcome feedback! To leave a comment click the icon in the upper right corner of the banner.
```

Comment:
Added the comment but I don't see an icon to click on. Maybe I'm missing something...